src/mesh/mesh_communication_global_indices.C

```diff
@@ -891,6 +927,19 @@ void MeshCommunication::find_global_indices (const Parallel::Communicator & comm
   libmesh_assert_equal_to(index_map.size(), n_objects);
 }
 #else // LIBMESH_HAVE_LIBHILBERT, LIBMESH_HAVE_MPI
+template <typename ForwardIterator>
+void MeshCommunication::find_local_indices (const libMesh::BoundingBox &,
+                                            const ForwardIterator & begin,
+                                            const ForwardIterator & end,
+                                            std::vector<dof_id_type> & index_map) const
```

Comment:
This needs to be fixed; I wrote the fallback code here before settling on the final API and I forgot to update it after.